modules/Kconfig.tinycbor

```diff
@@ -2,9 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config TINYCBOR
-	bool "tinyCBOR Support"
+	bool "DEPRECATED: tinyCBOR Support"
 	help
 	  This option enables the tinyCBOR library.
+	  Note: This module is deprecated, please use ZCBOR instead.
```

Comment:
Nit: I realize this is probably referring to CONFIG_ZCBOR, so it's up to you, but zcbor is styled in lower-case.

Reply:
Yes, this is about the Kconfig option.